### PR TITLE
Terminus equipment

### PR DESCRIPTION
**Pull request by kellan-simiotics**
Adding cli flows. Currently supports adding Terminus equipment and Lootboxes from config.

### CODE REVIEW

File: cli/enginecli/core.py [hunk 1]
```diff
@@ -516,13 +498,15 @@ def lootbox_gogogo(
...
-    terminus_contract.set_pool_controller(
-        admin_token_pool_id, lootbox_contract.address, tx_config
+    admin_terminus_contract.set_pool_controller(
+        admin_terminus_pool_id, lootbox_contract.address, tx_config
     )
 
     contracts = {
         "Lootbox": lootbox_contract.address,
-        "adminTokenPoolId": admin_token_pool_id,
+        "TerminusAddress": terminus_address,
```
Comment (zomglings) [inline cli/enginecli/core.py:507]: `TerminusAddress`, `AdminTerminusAddress`, and `AdminTokenPoolId` almost certainly do not belong at the root level of a dictionary called `contracts. I see that you are going off of the previous code, though, so no need to change this.

File: cli/enginecli/core.py [hunk 2]
```diff
@@ -516,13 +498,15 @@ def lootbox_gogogo(
     )
 
     print("Setting pool controller...")
-    terminus_contract.set_pool_controller(
-        admin_token_pool_id, lootbox_contract.address, tx_config
+    admin_terminus_contract.set_pool_controller(
+        admin_terminus_pool_id, lootbox_contract.address, tx_config
```
Comment (zomglings) [inline cli/enginecli/core.py:502]: This seems a bit suspect to me - does the Lootbox contract give a way of minting these badges to someone?

File: cli/enginecli/flows.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        equipment.create_pool_v2(
+            MAX_UINT, True, True, item["metadata"], transaction_config
+        )
+        pool_id = equipment.total_pools()
+        print(item["id"] + " => " + str(pool_id))
+
+    print("Ended with " + str(equipment.total_pools()) + " pools.")
+
+
+def handle_create_equippables_lootbox_from_config(args: argparse.Namespace) -> None:
```
Comment (zomglings) [inline cli/enginecli/flows.py:49]: Maybe it doesn't matter that they are `equippables`. I would do maybe `handle_create_items_lootbox_from_config`. The `items` denotes that they are Terminus tokens.

File: cli/enginecli/test_lootbox.py [hunk 4]
```diff
@@ -172,6 +203,33 @@ def _open_lootbox(self, account, lootboxId, count=None):
                     token_amount * count,
                 )
 
+            elif reward_type == 1:
```
Comment (zomglings) [inline cli/enginecli/test_lootbox.py:206]: You should check the `total_supply` on the pool before and after minting, as well.

**Review by zomglings — COMMENTED** [review]
Suggested a small addition to the mintable Terminus lootbox opening test.

File: cli/enginecli/core.py (hunk 2 above)
Comment (kellan-simiotics) [reply cli/enginecli/core.py:502]: There is a method on lootbox called grant-admin-role that requires this. See ControllableWithTerminus.sol.

File: cli/enginecli/flows.py (hunk 3 above)
Comment (kellan-simiotics) [reply cli/enginecli/flows.py]: Renamed clis.

File: cli/enginecli/test_lootbox.py (hunk 4 above)
Comment (kellan-simiotics) [reply cli/enginecli/test_lootbox.py:214]: Done

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**